### PR TITLE
Did a few things

### PR DESCRIPTION
**Pull request by IIodyne**
- Updated EveLiquidOceans to use hydroscoop instead of airscoop
- Fixed CCK category to include stock ore tanks
- Added RareMetals tank switch to ore tanks
- Added Kerbalism radiation emitters for nuclear engines
- Added KCT fuel type support
- Added Maine Coon jet's reactor to RR NERVAs, or KerbalAtomics reactor if installed
- Added B9 tank switch support for a bunch of tanks of various mods (more likely to come)

### CODE REVIEW

File: Extras/RR_KerbalismEmitters.cfg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//Add Kerbalism radiation emitters to nuclear engines
+@PART[RREngineNERVA*]:NEEDS[FeatureRadiation]:FOR[KerbalismDefault]
```
Comment (JadeOfMaar) [inline Extras/RR_KerbalismEmitters.cfg:2]: Never use `:FOR` inside a mod that you don't own. This tells MM to run this as part of the sequence of KerbalismDefault but also tells MM that KerbalismDefault is installed, and will cause patches depending on this in other mods to run, and can cause major conflicts.

MM does not respect NEEDS when FOR is declared. Replace it with BEFORE or AFTER.

File: Extras/MoreTankSwitches.cfg [hunk 2]
```diff
@@ -0,0 +1,1192 @@
+@PART[hydrogen-*]:NEEDS[B9PartSwitch,CryoTanks,RationalResourcesSquad]:AFTER[zzz_CryoTanks]
+{
+	// The CryoTanks themselves
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[fuelSwitch]]
+	{
+		SUBTYPE
```
Comment (JadeOfMaar) [inline Extras/MoreTankSwitches.cfg]: I purposely avoided adding heavy, non-cryo fuels as options (MonoPropellant, LiquidFuel) to the CryoTanks. I will not accept this bit, sorry, but I don't mind providing them in other tanks that aren't purpose made to hold cryo substances.
> * "Who's going to want that much toxic, heavy, inefficient fuel (Mono) on their spaceship?"
> * "When you install CryoTanks, the impression I get is that you're interested in Hydrolox capacity, not LiquidFuel. There are many mods for huge LFO tanks already."

File: Extras/RR_KerbalismEmitters.cfg (hunk 1 above)
Comment (IIodyne) [reply Extras/RR_KerbalismEmitters.cfg:2]: Gotcha; thought this was the case but then heard conflicting info so just went with first thing that worked.

File: Extras/MoreTankSwitches.cfg (hunk 2 above)
Comment (IIodyne) [reply Extras/MoreTankSwitches.cfg]: Understandable.

**Pull request merged**